### PR TITLE
payload-testing-ui: add deployment objects

### PR DESCRIPTION
**Pull request by bbguimaraes**
payload-testing-ui: add deployment objects

- service account
- cluster role to read the CRs
- cluster role binding for the SA
- deployment / service / route

---

ci-tools: add `payload-testing-ui` image

### CODE REVIEW

**Comment by bbguimaraes** [comment]
/hold

Requires https://github.com/openshift/ci-tools/pull/2452.

**Review by petr-muller — APPROVED** [review]

File: clusters/app.ci/payload-testing-ui/payload-testing-ui.yaml [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+    namespace: ${NAMESPACE}
+  userNames:
+  - system:serviceaccount:${NAMESPACE}:payload-testing-ui
+- apiVersion: v1
+  kind: Service
+  metadata:
+    namespace: ${NAMESPACE}
+    name: payload-testing-ui
+    labels:
+      app: prow
```
Comment (petr-muller) [inline clusters/app.ci/payload-testing-ui/payload-testing-ui.yaml:46]:
```suggestion
      app: pr-payload-tests
```

File: clusters/app.ci/payload-testing-ui/payload-testing-ui.yaml [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+      component: payload-testing-ui
+  spec:
+    ports:
+    - name: ui
+      port: 8000
+      protocol: TCP
+      targetPort: ui
+    selector:
+      app: prow
```
Comment (petr-muller) [inline clusters/app.ci/payload-testing-ui/payload-testing-ui.yaml:55]:
```suggestion
      app: pr-payload-tests
```

File: clusters/app.ci/payload-testing-ui/payload-testing-ui.yaml [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+      component: payload-testing-ui
+- apiVersion: route.openshift.io/v1
+  kind: Route
+  metadata:
+    namespace: ${NAMESPACE}
+    name: payload-testing-ui
+    labels:
+      app: prow
```
Comment (petr-muller) [inline clusters/app.ci/payload-testing-ui/payload-testing-ui.yaml:63]:
```suggestion
      app: pr-payload-tests
```

File: clusters/app.ci/payload-testing-ui/payload-testing-ui.yaml [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+    name: payload-testing-ui
+    annotations:
+      image.openshift.io/triggers: '[{"from":{"kind":"ImageStreamTag","name":"payload-testing-ui:latest"},"fieldPath":"spec.template.spec.containers[?(@.name==\"payload-testing-ui\")].image"}]'
+  spec:
+    replicas: ${{REPLICAS}}
+    strategy:
+      type: RollingUpdate
+    selector:
+      matchLabels:
+        app: prow
```
Comment (petr-muller) [inline clusters/app.ci/payload-testing-ui/payload-testing-ui.yaml:87]:
```suggestion
        app: pr-payload-tests
```

File: clusters/app.ci/payload-testing-ui/payload-testing-ui.yaml [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+        component: payload-testing-ui
+    template:
+      metadata:
+        labels:
+          app: prow
```
Comment (petr-muller) [inline clusters/app.ci/payload-testing-ui/payload-testing-ui.yaml:92]:
```suggestion
          app: pr-payload-tests
```

File: clusters/app.ci/payload-testing-ui/payload-testing-ui.yaml [hunk 6]
```diff
@@ -0,0 +1,128 @@
...
+    to:
+      kind: Service
+      name: payload-testing-ui
+- apiVersion: apps/v1
+  kind: Deployment
+  metadata:
+    namespace: ${NAMESPACE}
+    name: payload-testing-ui
+    annotations:
+      image.openshift.io/triggers: '[{"from":{"kind":"ImageStreamTag","name":"payload-testing-ui:latest"},"fieldPath":"spec.template.spec.containers[?(@.name==\"payload-testing-ui\")].image"}]'
```
Comment (petr-muller) [inline clusters/app.ci/payload-testing-ui/payload-testing-ui.yaml:80]: Please also give it a  `app: pr-payload-tests` label

**Review by petr-muller — APPROVED** [review]

**Pull request merged**